blog/docs/releases/announcing-knative-v1-12-release.md

```diff
@@ -0,0 +1,169 @@
+
+### 💫 New Features & Changes
+- Allow `shareProcessNamespace` to be set for a Knative Service (feature flag: `kubernetes.podspec-shareprocessnamespace`). Disabled by default. (#14454, @rhuss)
+- Internal encryption verifies a new SAN `kn-user-<ns>`. (#14452, @nak3)
+- ReadinessProbe with path contains a query string is supported now. (#14273, @nak3)
+- Support gRPC probe. (#14134, @seongpyoHong)
+- When `system-internal-tls` is enabled, queue-proxy mounts the certificate secret as projected-volume and automatically reloads the certificates on change. (#14189, @ReToCode)
+
+### 🐞Bug Fixes
+- Activator correctly propagates pod health when triggered by changes other than pod probes. (#14347, @arsenetar)
+- Activator no longer cancels all probes when one fails (#14303, @arsenetar)
+- Applied an upper bound to the statistics data read from the queue-proxy by the autoscaler. (#14523, @evankanderson)
+- Certificate generation errors are bubbled up to its parent Route. (#14496, @gabo1208)
+- Fix secure 'secure-pod-defaults' to work with restricted namespaces (#14363, @KauzClay)
+
+## Eventing
+
+[Release Notes](https://github.com/knative/eventing/releases/tag/knative-v1.12.0)
+
+### New Features
+
+- The `filters` field in Triggers is now **beta and enabled by default**
+  - New Event Filters are now only created once, rather than on each event (#7213, @Cali0707)
+  - The Any filter now dynamically optimizes the order of nested filters for optimal performance. (#7205, @Cali0707)
+  - The all filter now dynamically optimizes its ordering to improve performance (#7300, @Cali0707)
+  - The exact filter now uses less memory and is faster! (#7311, @Cali0707)
+  - The prefix filter just got a whole lot faster! (#7309, @Cali0707)
+  - The suffix filter is now faster! (#7312, @Cali0707)
+- OIDC authentication feature
+  - Add Audience field in CRDs (#7244, @xiangpingjiang)
+  - Expose OIDC audience of a Broker in its status (#7237, @creydr)
+  - Expose OIDC audience of an InMemoryChannel in its status (#7371, @creydr)
+  - Expose the APIServerSource OIDC service account name in the APIServerSource .status.auth.serviceAccountName (#7330, @Leo6Leo)
+  - Expose the PingSource OIDC service account name in the PingSource .status.auth.serviceAccountName (#7344, @Leo6Leo)
+  - Expose the SinkBinding OIDC service account name in the SinkBinding .status.auth.serviceAccountName (#7327, @rahulii)
+  - Expose the SubscriptionsOIDC service account name in the Subscriptions.status.auth.serviceAccountName (#7338, @xiangpingjiang)
+  - Expose the Triggers OIDC service account name in the Triggers .status.auth.serviceAccountName (#7299, @creydr)
+  - Mt-broker-ingress: verify the audience of the received JWT if OIDC authentication is enabled (#7336, @creydr)
+  - OIDC tokens are now cached to improve performance. (#7335, @Cali0707)
+- It is now possible to specify a subset of features in `config-features` without overriding default values (#7379, @pierDipi)
```

Comment:
@ReToCode the indentation of the points here didn't seem to render properly, I think it might need 4 spaces for mkdocs https://github.com/Python-Markdown/markdown/issues/3

Reply:
👍 Done